zerver/views/message_edit.py

```diff
@@ -112,19 +113,18 @@ def get_message_edit_history(
 PROPAGATE_MODE_VALUES = ["change_later", "change_one", "change_all"]
```

Comment:
This value is now unused; not sure what the best option is for cleaning that up; I feel like ultimately we will want that set of options to be a type and the `zerver/actions/message_edit.py` code to stop doing string comparisons on it.